docs/CI-CD.md

```diff
@@ -0,0 +1,93 @@
+# EVE CI/CD infrastructure
+
+EVE development happens on GitHub and we use GitHub's Actions as the core of our CI/CD infrastructure.
+Within GitHub actions we try to do as much as possible with the stock `ubuntu-20.04` build VM that
+gets instantiated from scratch every time jobs need to run as part of a given action. However, since
+we also have to produce ARM builds, we have a hosted runner tagged `ARM64` that we use for subset of
+the actions. EVE uses a standard OCI Registry (currently on [Docker HUB](https://hub.docker.com/r/lfedge/eve))
+to store all of its CI/CD build artifacts.
+
+Definitions of EVE CI/CD actions are available under [.github/workflows](../.github/workflows) folder
+and they currently include:
+
+* [Publish](https://github.com/lf-edge/eve/actions?query=workflow%3APublish) publishes various EVE build artifacts into an OCI Registry
+* [Pulls](https://github.com/lf-edge/eve/actions?query=workflow%3APRs) manages pull requests
+* [Apache Yetus](https://github.com/lf-edge/eve/actions?query=workflow%3A%22Apache+Yetus%22) runs static, lint-like [Apache Yetus](http://yetus.apache.org/) checks on EVE
+* [Eden](https://github.com/lf-edge/eve/actions?query=workflow%3AEden) runs integration tests from EVE's sister project [Eden](https://github.com/lf-edge/eden)
+
+Since EVE community generally frowns on branch-heavy development most of the descriptions below
+will only apply to the master branch (unless stated otherwise).
+
+## Publish
+
+This is the only action that uses both x86 and ARM64 runners. It gets triggered on either updates
+to the `master` branch or creation of release tags. Publish produces a set of tagged artifacts and
+pushes them into EVE's OCI Registry containers. Logic responsible for figuring out correct tags
```

Comment:
"OCI Registry containers" - should that not be "OCI Registry repositories"?

Reply:
done